pkg/api/testapi/testapi.go

```diff
@@ -55,9 +55,10 @@ var (
 )
 
 type TestGroup struct {
-	externalGroupVersion unversioned.GroupVersion
-	internalGroupVersion unversioned.GroupVersion
-	internalTypes        map[string]reflect.Type
+	// the first element in the group is meant to be the preferred version
```

Comment:
Thanks for dealing with this file. I think it needs some help in the future but I won't ask you do to that.